en/docs/setup/changing-to-mssql.md

```diff
@@ -181,11 +181,11 @@ Apart from above basic configurations WSO2 Identity Server supports advanced dat
     </tbody>
     </table>
 
-    !!! info 
-        For more information on other parameters that can be defined in
-        the `<IS_HOME>/repository/conf/deployment.toml` file, see [Tomcat
-        JDBC Connection
-        Pool](http://tomcat.apache.org/tomcat-7.0-doc/jdbc-pool.html#Tomcat_JDBC_Enhanced_Attributes).
+!!! info 
+    For more information on other parameters that can be defined in
+    the `<IS_HOME>/repository/conf/deployment.toml` file, see [Tomcat
+    JDBC Connection
+    Pool](http://tomcat.apache.org/tomcat-7.0-doc/jdbc-pool.html#Tomcat_JDBC_Enhanced_Attributes).
```

Comment:
Shall we point to Tomcat 9 document? Do this in all applicable places

Reply:
Fixed : 3714c5859a2c616edf12d1dc94282501e2d97f71